internal/apex/apex.go

```diff
@@ -265,7 +265,7 @@ func (ax *Apex) Run() error {
 		if err := ax.Reconcile(user.ZoneID, false); err != nil {
 			// TODO: Add smarter reconciliation logic
 			// to handle disconnects and/or timeouts etc...
-			return err
+			ax.logger.Error(err)
```

Comment:
How about adding a prefix to the log message that makes it a bit more clear what the context of the error is?

```suggestion
			ax.logger.Errorf("Failed to reconcile state with the apex API server: ", err)
```